src/Type/Php/ClassExistsFunctionTypeSpecifyingExtension.php

```diff
@@ -54,6 +55,17 @@ public function specifyTypes(FunctionReflection $functionReflection, FuncCall $n
 			);
 		}
 
+		if ($functionReflection->getName() === 'enum_exists') {
+			return $this->typeSpecifier->create(
```

Comment:
Please less code duplication here. The only thing different is the type argument.